### PR TITLE
Update clone instructions for the public repo

### PR DESCRIPTION
**Pull request by embeddery**
## Description

Per title.

## Checklist
- [x] Investigated and inspected CI test results

## Testing Performed

n/a just some docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,25 +104,17 @@ The following tools are necessary to test code and build image(s):
...
-
-# Instruct Go to bypass the Go package proxy for our private dependencies
-$ go env -w GOPRIVATE=github.com/stackrox
-
-$ cd $GOPATH
-$ mkdir -p src/github.com/stackrox
-$ cd src/github.com/stackrox
-$ git clone git@github.com:stackrox/stackrox.git
+cd $GOPATH
+mkdir bin pkg src
```
Comment (ehashman) [inline README.md:114]: Suggest `mkdir -p` as this will fail if the directories already exist

**Review by ehashman — APPROVED** [review]
/lgtm

**Comment by embeddery** [comment]
Can a maintainer run CI plz?

**Comment by gavin-stackrox** [comment]
Cannot merge without CI :(

**Pull request closed without merging**